### DISCUSSION TITLE
Feedbacks after months of use, and correct way to update

### DISCUSSION THREAD

**Discussion by beber666** [mikebrady/shairport-sync#1789] [Q&A] [answered]
Hello

I'm using your solution in my house with multiple Orange PI Zero's & Airplay 2.
Your software is the better I've found, thanks again for everything !

Could you please give me the correct way to update ? 

Thank you !

Bertrand

**Comment by MSL-DA** [comment, marked as answer]
The easy way:
https://github.com/mikebrady/shairport-sync/discussions/1566#discussion-4518475

**Reply by beber666** [reply]
Thank you !
